smoke-tests/test_helpers/utilities.bash

```diff
@@ -20,7 +20,7 @@ span_attributes_for() {
 #   $3 - attribute key
 #   $4 - attribute type
 attribute_for_span_key() {
-	attributes_from_span_named $1 $2 | \
+	attributes_from_span_named "$1" "$2" | \
```

Comment:
We're emitting spans named `View Render` so these args need to be quoted for our utils to work correctly.